### DISCUSSION TITLE
WPF Support

### DISCUSSION THREAD

**Discussion by prestsauce** [Equipple/vs-templates-revit-addin#10] [Q&A]
Hello @salaros 

Thanks for providing this- it's been very insightful. I'm still very new to MSBuild, and I've been having a lot of difficulty adapting the template to support WPF forms. 

Is there any advice or direction you can give?

Thanks.

**Comment by salaros** [comment]
Hi @prestsauce you are right UI support wasn't good, please try downloading the latest version
https://github.com/salaros/vs-templates-revit-addin/archive/master.zip

I'm really willing to hear from you after you try this new version!

**Reply by prestsauce** [reply]
Sorry for the very late reply - I haven't been coding much lately. I can't wait to take a look! I'll get back to you once I get to try it out. Thanks!

**Reply by Marcelo-Romero** [reply]
Hi @salaros , First of all, thank you for this amazing work you did creating these templates! I'm having just one issue. I downloaded the latest version of the template and I'm having a hard time creating a simple Windows Form. I'm using Visual Studio 2022 and when creating the Form, there are no tools in the toolbox. When I click on "Show All" they show up but they are all greyed out. Have you had this issue before? Am I missing something? Appreciate your time.

**Reply by Marcelo-Romero** [reply]
Nevermind. Once I changed the Targeting Framework to Revit 2023 they showed up for some reason. All good now!
